src/dota2/decoders.ts

```diff
@@ -2,6 +2,7 @@ import {
   IAbility,
   IBuilding,
   IDraft,
+  IEvent,
```

Comment:
I dislike that this is called `IEvent` because there are so many different types of events, so it may be confusing.

However, this is what the Valve calls it. 

Open to any naming feedback